packages/shopify-codemod/transforms/function-to-arrow.js

```diff
@@ -11,7 +11,19 @@ export default function functionToArrow({source}, {jscodeshift: j}, {printOption
     return !isMember(path) && !containsThisExpression(path);
   }
 
-  return j(source)
+  const root = j(source);
+
+  root
```

Comment:
Should we use `const sourceAST` as a convention for caching the AST, since other files are currently using `sourceAST`?